code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -223,6 +223,30 @@
 		M.adjustFireLoss(-0.5*REM)
 	..()
 	return
+	
+/datum/reagent/medicine/mine_salve
+	name = "Miner's Salve"
+	id = "mine_salve"
+	description = "Slowly heals burn and brute damage, and causes subject to believe they are fully healed."
+	reagent_state = LIQUID
+	color = "#6D6374"
+	metabolization_rate = 0.4 * REAGENTS_METABOLISM
+
+/datum/reagent/medicine/mine_salve/on_mob_life(mob/living/M)
+	if(istype(M, /mob/living/carbon))
```

Comment:
There's an IsCarbon() check which does the same as this and is better to use for some reason.